pinot-query-planner/src/test/resources/queries/PinotHintablePlans.json

```diff
@@ -1,6 +1,39 @@
 {
   "pinot_hint_option_tests": {
     "queries": [
+      {
+        "description": "SELECT * inner join with filter on one table",
+        "sql": "EXPLAIN PLAN FOR SELECT /*+ joinOptions(is_colocated_by_join_keys='true') */ * FROM a JOIN b ON a.col1 = b.col2 WHERE a.col3 >= 0",
+        "output": [
+          "Execution Plan",
+          "\nLogicalJoin(condition=[=($0, $6)], joinType=[inner])",
+          "\n  LogicalExchange(distribution=[single])",
+          "\n    LogicalFilter(condition=[>=($2, 0)])",
+          "\n      LogicalTableScan(table=[[a]])",
+          "\n  LogicalExchange(distribution=[single])",
+          "\n    LogicalTableScan(table=[[b]])",
+          "\n"
+        ]
+      },
+      {
+        "description": "Inner join with group by",
```

Comment:
How is the hint been used when you have two joins and you want to singleton on just one of them or both of them?